### DISCUSSION TITLE
Create a README.md file for this repository?

### DISCUSSION THREAD

**Discussion by oood** [abbrcode/abbreviations-in-code#91] [Ideas]
I think we need a big button to make it easier for people to find where "discussions" are.

Also I recommend closing unused tabs under [the repository](https://github.com/abbrcodes/discussions), like issues.

**Comment by oood** [comment]
We may need to find a image of the button and then hyperlink it in the readme file.

Image need to be copyright-free, of course.

**Reply by T1xx1** [reply]
> We may need to find a image of the button and then hyperlink it in the readme file.
> 
> Image need to be copyright-free, of course.

You mean having a img like button in the readme so ref discussion?

**Reply by oood** [reply]
> You mean having a img like button in the readme so ref discussion?

Yes, because imgs can be more visual than words.

**Comment by T1xx1** [comment]
> I think we need a big button to make it easier for people to find where "discussions" are.

Yes. I was thinkg of the special .github repo so when someone goes into the org account the readme shows under it

> Also I recommend closing unused tabs under the repository, like issues.

Are issues aviable in orgs? I don't see them

![Screenshot (114)](https://user-images.githubusercontent.com/69322987/223539750-391d6b9f-a48a-4eb9-8d90-f1bd1b7fc934.png)

**Reply by T1xx1** [reply]
Ohh you mean in the repo. Sorry was browsing from the org profile. Is there a way to close issues (i'm checking).

**Reply by oood** [reply]
> Yes. I was thinkg of the special .github repo so when someone goes into the org account the readme shows under it

Actually it's already linked to the organization, but something like a user page would be cool.

https://github.com/orgs/abbrcodes/discussions

> Are issues aviable in orgs? I don't see them

github says: yes

> Create Member README
>
> Create a private repository called .github-private. Once created, add a /profile/README.md. The README will appear on your organization's profile, visible only to members of the organization.
> 
> [Read documentation ](https://docs.github.com/en/organizations/collaborating-with-groups-in-organizations/customizing-your-organizations-profile)

Also, you can switch to a public repository, visible to everyone.

To the right of the [organization](https://github.com/abbrcodes), if you're using a desktop browser

**Reply by T1xx1** [reply]
> > Yes. I was thinkg of the special .github repo so when someone goes into the org account the readme shows under it
> 
> Actually it's already linked to the organization, but something like a user page would be cool.
> 
> https://github.com/orgs/abbrcodes/discussions

Yes I've pinned it before but it not like having a big DISCUSSION btn.

> > Are issues aviable in orgs? I don't see them
> 
> github says: yes

Yep find them. I was in the wrong repo.


> 
> > Create Member README
> > Create a private repository called .github-private. Once created, add a /profile/README.md. The README will appear on your organization's profile, visible only to members of the organization.
> > [Read documentation ](https://docs.github.com/en/organizations/collaborating-with-groups-in-organizations/customizing-your-organizations-profile)
> 
> Also, you can switch to a public repository, visible to everyone.

But they are 2 different repos.

We can have 2 .github (let's call them profiles)
.github profile available to everyone
.github-private profile only for org members.

I know that because I've already used them before.

**Reply by oood** [reply]
That's right, I've seen a lot of user page customizations, never tried it.

> I know that because I've already used them before.

Would you like to follow up on this request?

**Reply by T1xx1** [reply]
Yep, I'm making one commit to fix a pr that had conflicts. I'll create the .github repo and I'll leave it like that for tomorrow, they requested me to play videogames hahaha.

**Reply by oood** [reply]
> Yep, I'm making one commit to fix a pr that had conflicts. I'll create the .github repo and I'll leave it like that for tomorrow, they requested me to play videogames hahaha.

no problem! Feeling free to do this, we're going pretty fast right now.

Leave Brevis to me and I'll get it done tonight.

**Comment by oood** [comment]
UPDATE: The request has been added to Brevis.

https://github.com/orgs/abbrcodes/projects/1/views/1?pane=issue&itemId=22307888

**Comment by oood** [comment]
UPDATE:

* Turn off the Actions tab under [this repository](https://github.com/abbrcodes/discussions), as this repository probably won't need to run any code.

* The fork option also turned off, as the repository should not be used for code.

**Reply by T1xx1** [reply]
> UPDATE:
> 
> * Turn off the Actions tab under this repository, as this repository probably won't need to run any code.
> * The fork option also turned off, as the repository should not be used for code.

Ok.

Meanwhile. .github created, .github-private created and i've also renamed main the "main" branch of abbr in code.

**Reply by oood** [reply]
> Meanwhile. .github created, .github-private created and i've also renamed main the "main" branch of abbr in code.

that would be great! 

It seems that the name of the default branch can be defined in [the org settings page](https://github.com/organizations/abbrcodes/settings/profile), I may have to check that later.

**Reply by T1xx1** [reply]
It's ok it's already set to main.

**Comment by T1xx1** [comment]
[README](https://github.com/abbrcode?view_as=public)
